sdk/core/azure-core/src/main/java/com/azure/core/spatial/PointGeometry.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.spatial;
+
+/**
+ * Represents a geometric point.
+ */
+public final class PointGeometry extends Geometry {
+    private final GeometryPosition position;
+
+    /**
+     * Constructs a geometric point.
+     *
+     * @param position The {@link GeometryPosition geometric position} of the point.
+     */
+    public PointGeometry(GeometryPosition position) {
+        this(position, null);
+    }
+
+    /**
+     * Constructs a geometric point.
+     *
+     * @param position The {@link GeometryPosition geometric position} of the point.
+     * @param properties Additional properties of the geometric point.
+     */
+    public PointGeometry(GeometryPosition position, GeometryProperties properties) {
```

Comment:
Does search need to provide specific serializer/deserializer for coordinate system?

Reply:
Search shouldn't need to have a custom serializer/deserializer as they should be using the GeoJSON standard.

Reply:
Search may need custom handling when using `PointGeometry` when setting an OData `Edm.GeographyPoint`.